docs/HANDLED_RULES.md

```diff
@@ -141,6 +141,19 @@ Example:
 
 -----
 
+#### JEE applications should not "getClassLoader" ([Sonar Rule 3032](https://rules.sonarsource.com/java/RSPEC-3032))
+
+Using the standard `getClassLoader()` may not return the right class loader in Java Enterprise Edition context. Instead, `getClassLoader()` usage such as `this.getClass().getClassLoader()` and `Dummy.class.getClassLoader()` should be replaced by `Thread.currentThread().getClassLoader()`. In particular, such replacement only occurs if the `.java` files use the `javax` package in its imports.
```

Comment:
"Thread.currentThread().getClassLoader()" -> "Thread.currentThread().getContextClassLoader()"

"the `.java` files use" -> "the `.java` file uses"

Reply:
Henry, you didn't fix everything here. See that you say "its imports" thus you need to replace "the .java files use" to "the .java file uses".

Reply:
done :) .